srcpkgs/ly/files/ly/conf

```diff
@@ -0,0 +1,7 @@
+BAUD_RATE=38400
+TERM_NAME=linux
+TTY=7
```

Comment:
Perhaps leave a comment here pointing to the service conf?

Reply:
Changed!